conductr_cli/conductr_restore.py

```diff
@@ -0,0 +1,114 @@
+import json
+import os
+import shutil
+import tempfile
+
+import io
+import copy
+import logging
+
+from conductr_cli import control_protocol, bundle_utils, validation, conduct_load, bundle_scale
+from conductr_cli.bundle_core_info import BundleCoreInfo
+
+
+@validation.handle_connection_error
+@validation.handle_http_error
+@validation.handle_wait_timeout_error
+def restore(args):
+    log = logging.getLogger(__name__)
+    destination_bundles = control_protocol.get_bundles(args)
+    destination_bundles_info = BundleCoreInfo.from_bundles(destination_bundles)
+
```

Comment:
Right around here I would specialize handling for `stdin`. Ideally we'd like this syntax to be supported: `conduct backup | conduct restore`

To do this, `args.backup` should be optional IFF not on a TTY - this signifies stdin should be read from. Additionally, if `args.backup` equals `-` this should also signify to read from `stdin`.